.ci/release/Jenkinsfile

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+pipeline {
+  agent { label 'linux && immutable' }
+  environment {
+    REPO = 'apm-agent-java'
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    NOTIFY_TO = credentials('notify-to')
+    DOCKERHUB_SECRET = 'secret/apm-team/ci/elastic-observability-dockerhub'
+    ELASTIC_DOCKER_SECRET = 'secret/apm-team/ci/docker-registry/prod'
+    NEXUS_SECRET = 'secret/apm-team/ci/nexus'
+    MAVEN_CONFIG = '-Dmaven.repo.local=.m2'
```

Comment:
nit:
maybe DEFAULT_MAVEN_CONFIG ?

Reply:
I said nothing, that's how it's defined in the other Jenkinsfile!

Reply:
Aye. I just moved it from there. :)